scapy/arch/windows/compatibility.py

```diff
@@ -9,6 +9,10 @@
 
 from scapy.arch.consts import LOOPBACK_NAME
 from scapy.config import conf,ConfClass
+from scapy.error import warning, Scapy_Exception, log_runtime
+from scapy.data import *
+from scapy.arch import pcapdnet
+from scapy.arch.pcapdnet import *
 
```

Comment:
I do not really understand why all of this is useful, wasn't it already working ?